template.php

```diff
@@ -1786,18 +1786,24 @@ function ec_resp_nexteuropa_multilingual_language_list(array $variables) {
  *   The internal path being linked to.
  * @param array $options
  *   An associative array of additional options.
+ * @param array $all_paths
+ *   An associative array of paths keyed by their language code.
+ *   If it is not empty, the array items will be used to generate the links of
+ *   language list.
  *
  * @return string
  *   Formatted HTML column displaying the list of provided languages.
  */
-function _ec_resp_nexteuropa_multilingual_language_list_column($languages, $path, $options) {
+function _ec_resp_nexteuropa_multilingual_language_list_column($languages, $path, $options, $all_paths) {
   $content = '<div class="col-sm-6">';
   foreach ($languages as $language) {
     $options['attributes']['lang'] = $language->language;
     $options['attributes']['hreflang'] = $language->language;
     $options['attributes']['class'] = 'btn splash-page__btn-language';
     $options['language'] = $language;
-    $content .= l($language->native, $path, $options);
+
+    $translated_path = (!empty($all_paths)) ? $all_paths[$language->language] : $path;
```

Comment:
If the content is translated but not in $language->language, $all_paths[$language->language] is not there while $all_paths is not empty.